  RuntimeError(const RuntimeError &rerr) : std::runtime_error(rerr) {}